client/components/TestRun/index.jsx

```diff
@@ -90,19 +133,46 @@ class TestRun extends Component {
                 </React.Fragment>
             );
             if (!this.state.runComplete) {
-                content = (
-                    <React.Fragment>
-                        <iframe
-                            src={`/aria-at/${git_hash}/${
-                                tests[this.state.currentTestIndex - 1].file
-                            }?at=${at_key}`}
-                            id="test-iframe"
-                        ></iframe>
-                        <Button variant="primary" onClick={this.nextTest}>
-                            Next test
-                        </Button>
-                    </React.Fragment>
-                );
+                if (!test.result) {
+                    content = (
+                        <React.Fragment>
+                            <iframe
+                                src={`/aria-at/${git_hash}/${
+                                    tests[this.state.currentTestIndex - 1].file
+                                }?at=${at_key}`}
+                                id="test-iframe"
+                                ref={this.testIframe}
+                            ></iframe>
+                            <Button
+                                variant="primary"
+                                onClick={this.handleSkipTestClick}
+                            >
+                                Skip to next test
+                            </Button>
+                            <Button
+                                variant="primary"
+                                onClick={this.handleSaveResultClick}
+                            >
+                                Save results and go to next test
+                            </Button>
```

Comment:
Do all of these buttons need an aria-label?

Reply:
same as other button